packages/evm/contracts/disputes/DisputeManager.sol

```diff
@@ -1126,34 +1104,102 @@ contract DisputeManager is ControlledRecoverable, ICRVotingOwner, IDisputeManage
     }
 
     /**
-    * @dev Internal function to check if a certain dispute exists, it reverts if it doesn't
-    * @param _disputeId Identification number of the dispute to be checked
+    * @dev Internal function to check if a certain dispute round exists, it reverts if it doesn't
+    * @param _dispute Dispute to be checked
+    * @param _roundId Identification number of the dispute round to be checked
     */
-    function _checkDisputeExists(uint256 _disputeId) internal view {
+    function _checkRoundExists(Dispute storage _dispute, uint256 _roundId) internal view {
```

Comment:
Maybe this could be inlined into a `_getDisputeRound()`? All the usages seem to just be interested in the round :).